### ISSUE TITLE
multiple outputs

### ISSUE THREAD

**Issue by bernt-matthias** [bbuchfink/diamond#218]
is it possible to create multiple types of output in one run, eg. blast tabular + Taxonomic classification (102)?

Btw. 102 is not yet included in `diamond help`.

**Comment by bbuchfink** [comment]
No, the DAA format is supposed to provide that feature, so you can
first generate a DAA file and then use it to generate any other
format. Although the DAA format does not support the taxonomic format
at the moment, I still need to implement that.

**Comment by bernt-matthias** [comment]
Thanks for the explanation. I will leave the issue open.
